src/DataBuilder.php

```diff
@@ -790,20 +792,44 @@ protected function getRequestBody()
      */
     protected function getUserIp()
     {
-        if (!isset($_SERVER) || !$this->captureIP) {
+        if (!isset($_SERVER) || $this->captureIP === false) {
             return null;
         }
+        
+        $ip = null;
+        
         $forwardFor = isset($_SERVER['HTTP_X_FORWARDED_FOR']) ? $_SERVER['HTTP_X_FORWARDED_FOR'] : null;
         if ($forwardFor) {
             // return everything until the first comma
             $parts = explode(',', $forwardFor);
-            return $parts[0];
+            $ip = $parts[0];
         }
         $realIp = isset($_SERVER['HTTP_X_REAL_IP']) ? $_SERVER['HTTP_X_REAL_IP'] : null;
         if ($realIp) {
-            return $realIp;
+            $ip = $realIp;
+        }
+        $ip = isset($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : null;
+        
+        if ($this->captureIP === DataBuilder::ANONYMIZE_IP) {
+            
+            if (filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
+                
+                $parts = explode('.', $ip);
+                $ip = $parts[0] . '.' . $parts[1] . '.' . $parts[2] . '.0/24';
```

Comment:
@ArturMoczulski I just noticed that if you replace the last 8 bits of human-readable ip with "0/24", rollbar won't index it and you won't be able to see other errors logged from the same ip. My suggestion would be dropping the "/24" mask and leaving only the "0" at the end. Please see attached image where the addressed with the mask are not clickable.
![image](https://user-images.githubusercontent.com/1895494/40241473-7f25cd08-5a91-11e8-9749-f534cd01cb7a.png)

Reply:
Yeah I forgot to mention this before this got merged, we need to change the .0/24 to .0 That’s what I ended up doing for the rest of the SDKs